include/swift/AST/Decl.h

```diff
@@ -5714,14 +5714,17 @@ class AbstractFunctionDecl : public GenericContext, public ValueDecl {
   /// Note that the body was skipped for this function.  Function body
   /// cannot be attached after this call.
   void setBodySkipped(SourceRange bodyRange) {
-    assert(getBodyKind() == BodyKind::None);
+//    assert(getBodyKind() == BodyKind::None ||
```

Comment:
This needs a FIXME, I'd like to change the assert to the commented version once we enable delayed parsing everywhere.